README.md

```diff
@@ -15,11 +15,17 @@ cd libioc
 make install
 ```
 
-At the current time libiocage is not packaged or available in FreeBSD ports.
+The default Python version is 3.7. If you intend to run libioc from another version, please specify it during the installation:
+
+```sh
+make PYTHON=python3.8 install
+```
+
+At the current time libioc is not packaged or available in FreeBSD ports.
```

Comment:
this is not entirely true…
https://www.freshports.org/devel/py-libioc it just… *still* doesn't have all the right dependencies…

Reply:
Well, let's tackle this at a different time. Would you open an Issue as a reminder? 💜